workshops/templates/workshops/profileupdaterequest.html

```diff
@@ -118,6 +109,16 @@
         {% endif %}
         </td>
     </tr>
+    <tr><th colspan="3" class="text-center">Additional information (not being saved to the database)</th></tr>
```

Comment:
Because neither ORCID ID or Occupation is being stored in Person record.

The "Has instructor badge" is rather a property that can only be calculated from Person record.